doc/build-windows.md

```diff
@@ -68,7 +68,7 @@ To build executables for Windows 64-bit, install the following dependencies:
 Then build using:
 
     cd depends
-    make HOST=x86_64-w64-mingw32
+    make HOST=x86_64-w64-mingw32 PATH=$(getconf PATH)
```

Comment:
The `make` command worked successfully on WSL 14.04 using this command. It could potentially break things if people are relying on dependencies outside of `/bin:/usr/bin` path, but I didn't experience any issues building with only the required dependencies.